addons/mail/static/src/models/messaging/messaging_tests.js

```diff
@@ -1,64 +1,28 @@
 odoo.define('mail/static/src/models/messaging/messaging_tests.js', function (require) {
 'use strict';
 
-const {
-    afterEach: utilsAfterEach,
-    beforeEach: utilsBeforeEach,
-    start: utilsStart,
-} = require('mail/static/src/utils/test_utils.js');
+const { afterEach, beforeEach, start } = require('mail/static/src/utils/test_utils.js');
 
 QUnit.module('mail', {}, function () {
 QUnit.module('models', {}, function () {
 QUnit.module('messaging', {}, function () {
 QUnit.module('messaging_tests.js', {
     beforeEach() {
-        utilsBeforeEach(this);
+        beforeEach(this);
 
         this.start = async params => {
-            if (this.widget) {
-                this.widget.destroy();
-            }
-            const { env, widget } = await utilsStart(Object.assign({}, params, {
+            const { env, widget } = await start(Object.assign({}, params, {
                 data: this.data,
             }));
             this.env = env;
             this.widget = widget;
         };
     },
     afterEach() {
-        utilsAfterEach(this);
-        this.env = undefined;
-        if (this.widget) {
-            this.widget.destroy();
-            this.widget = undefined;
-        }
+        afterEach(this);
     },
 });
 
-QUnit.test('currentPartner initialized from session', async function (assert) {
-    assert.expect(5);
-
-    await this.start();
-
-    assert.ok(this.env.messaging.currentPartner);
-    assert.strictEqual(
-        this.env.messaging.currentPartner.id,
-        this.env.session.partner_id
-    );
-    assert.strictEqual(
-        this.env.messaging.currentPartner.name,
-        this.env.session.name
-    );
-    assert.strictEqual(
-        this.env.messaging.currentPartner.display_name,
-        this.env.session.partner_display_name
-    );
-    assert.strictEqual(
-        this.env.messaging.currentPartner.user.id,
-        this.env.session.uid
-    );
-});
-
```

Comment:
They are now returned by `init_messaging` instead of read from the session. It's to have the mock server as single source of truth in tests, to avoid having `this.data.currentPartnerId` messing up with the session values or vice-versa.
Ideally session should be computed from that value too, but this change would pretty much require moving a lot of what I have done here in /web, which I would consider out of scope.

Reply:
Would be nice to convert this test that checks `this.env.messaging.currentPartner` is well set up from mock server data (hard-coded `this.data.currentPartnerId` is acceptable).

Reply:
These data are deleted during init because they are fake data...